scenarios/Assistants/api-in-a-box/failed_banks/assistant-failed_banks.ipynb

```diff
@@ -4,9 +4,57 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Failed Banks Information Assistant\n",
+    "# Failed Banks Information Assistant"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Objective\n",
     "\n",
-    "### Load the required libraries"
+    "{{TODO: Add a one or two sentences describing the learning outcomes for this notebook}}\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- {{TODO: Add short descriptions and links to documentation for Azure AI services used}}\n"
```

Comment:
Needs objective + link to documentation

Reply:
# Failed Banks Information Assistant

## Objective

This notebook demonstrates the following:

- Using Assistant tools Code Interpreter and Function calling, this bot can get a CSV file, gather a list of failed banks by state, and generate a chart to visually represent the data.

This tutorial uses the following Azure AI services:
- Access to Azure OpenAI Service - you can apply for access [here](https://aka.ms/oai/access)
- Azure OpenAI service - you can create it from instructions [here](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource)
- Azure OpenAI Studio - go to [https://oai.azure.com/](https://oai.azure.com/) to work with the Assistants API Playground
- A connection to the Azure OpenAI Service with a [Key and Endpoint](https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart)

Reference:
- Learn more about how to use Assistants with our [How-to guide on Assistants](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant).
- [Assistants OpenAI Overview](https://platform.openai.com/docs/assistants/overview)

## Time
You should expect to spend 5-10 minutes running this sample.

Reply:
Resolved by https://github.com/Azure-Samples/azureai-samples/pull/59/commits/1e8d90231a99d4fc567c73b26279ba4a8484a754